src/main/frontend/modules/outliner/pipeline.cljs

```diff
@@ -2,12 +2,12 @@
   (:require [frontend.modules.datascript-report.core :as ds-report]
             [frontend.modules.outliner.file :as file]
             [frontend.db :as db]
-            [frontend.state :as state]
             [frontend.util :as util]
             [frontend.debug :as debug]))
 
+;; TODO: Do something or remove
```

Comment:
This is called from invoke-hooks and doesn't do anything. I don't know if this is a bug or just a todo for later

Reply:
It's a todo for later, we can comment out it now since there's no implementation yet.